src/database/rulesConfig.ts

```diff
@@ -0,0 +1,50 @@
+import * as Config from "../config";
+import { FirebaseError } from "../error";
+
+export interface RulesInstanceConfig {
+  instance: string;
+  rules: string;
+}
+
+interface DatabaseConfig {
+  rules: string;
+  target?: string;
+  instance?: string;
+}
+
+export function getRulesConfig(projectId: string, options: any): RulesInstanceConfig[] {
+  // TODO(samstern): Use the real config type here
```

Comment:
If I do `const config: Config = options.config` ts tries to tell me that there's no `.get(...)` function so I assume my import is wrong?

Reply:
Maybe do `options.config as Config`? Or, you can use a type guard: https://www.typescriptlang.org/docs/handbook/advanced-types.html#type-guards-and-differentiating-types

Reply:
I have tried about 50 different kinds of imports, declarations, and guards.  For some reason TS can see the properties of config from the constructor (`.data`, etc) but not any of the functions.  I declare defeat.

Reply:
🤷 - sometimes the easiest way is to `any` it